tests/metagen/typegraphs/sample/ts/client.ts

```
@@ -207,17 +208,14 @@ export class MutationNode<Out> {
   }
 }
 
-type SelectNodeOut<T> = T extends QueryNode<infer O> | MutationNode<infer O>
-  ? O
+type SelectNodeOut<T> = T extends QueryNode<infer O> | MutationNode<infer O> ? O
```

Biome (1.9.4)

[error] Shouldn't redeclare 'O'. Consider to delete it or rename it.

'O' is defined here:


(lint/suspicious/noRedeclare)